ale_linters/r/languageserver.vim

```diff
@@ -0,0 +1,26 @@
+" Author: Eric Zhao <21zhaoe@protonmail.com>
+" Description: Implementation of the Language Server Protocol for R.
+
+call ale#Set('r_languageserver_cmd', 'languageserver::run()')
+call ale#Set('r_languageserver_options', {})
```

Comment:
I'll rename the option to `_config`. I like to use `_config` for LSP configuration, and `_options` for command line options.